packages/console/src/pages/SignInExperience/components/LanguagesForm.tsx

```diff
@@ -60,14 +73,16 @@ const LanguagesForm = ({ isManageLanguageVisible = false }: Props) => {
             : t('sign_in_exp.others.languages.default_language_description_fixed')}
         </div>
       </FormField>
-      <CustomPhrasesContextProvider value={customPhrasesContext}>
+      <LanguageEditorContextProvider value={languageEditorContext}>
```

Comment:
can we put this provider inside "ManageLanguageModal"

Reply:
Add an issue to refactor this.
[Extract ManageLanguageModal into a component](https://linear.app/silverhand/issue/LOG-4321/extract-managelanguagemodal-into-a-component)